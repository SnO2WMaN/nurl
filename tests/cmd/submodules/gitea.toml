args = [
    "https://repo.palemoon.org/MoonchildProductions/Pale-Moon",
    "31.0.0_Release",
    "--submodules",
]
