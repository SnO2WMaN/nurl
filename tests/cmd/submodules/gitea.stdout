fetchFromGitea {
  domain = "repo.palemoon.org";
  owner = "MoonchildProductions";
  repo = "Pale-Moon";
  rev = "31.0.0_Release";
  hash = "sha256-fIQAQCtjA/9Otft3e9Z4xWgE09sqsdArYQtZqmEgfTc=";
  fetchSubmodules = true;
}